styles/designs/carnival/parts/_columns-components.scss

```diff
@@ -158,6 +182,22 @@ $Columns--FourColumnsSixRatioOne: (
     )
 );
 
+$Columns--FourColumnsSixRatioOneA: (
+    _name: "FourColumnsSixRatioOne",
+    _subselector: " [data-columns='4'][data-ratio='1,1,2,2'] col[data-width='1*']",
+    _properties: (
+        width: column-width(4,6,1),
+    )
+);
+
+$Columns--FourColumnsSixRatioTwoA: (
+    _name: "FourColumnsSixRatioOne",
+    _subselector: " [data-columns='4'][data-ratio='1,1,2,2'] col[data-width='2*']",
+    _properties: (
+        width: column-width(4,6,1),
+    )
+);
```

Comment:
wonder if this should be `TwoA` or just `Two` -- I don't see a `FourColumnsSixRatioTwo` anywhere ...

also wondering if with should be `(4,6,2)` instead of `(4,6,1)`

Reply:
I can change it to `Two`, however I named it TwoA to signify it's pairing with `FourColumnsSixRatioOneA`. 
(since the have the same data ratio)